examples/lifecycleDemo.html

```diff
@@ -136,40 +188,76 @@ <h1>Lifecycle Events Log:</h1>
                     // Simulating bootstrap delay of 500ms
                     window.setTimeout(() => {
                         myClientApp.lifecycle.bootstrapped();
-
-                        myClientApp.alerting.showToastPopup(
-                            lifecycleStatusMessageTitle,
-                            'Bootstrap Complete (500ms delay)', {
-                                id: lifecycleStatusMessageId,
-                                type: 'success'
-                            }
-                        );
+                        if (requireManualToastDismissal) {
+                            myClientApp.alerting.showToastPopup(
+                                lifecycleStatusMessageTitle,
+                                `Bootstrap Complete (500ms delay):lifecycleStatusMessageId-bootstrap-${count}`, {
```

Comment:
Did you mean to use lifecycleStatusMessageId as a variable or string here?

You could pull this all out to a utility method passing desc base, type, and requireManualToastDismissal.

```js
_showLifecycleToast('Bootstrap Complete (500ms delay)', 'success', requireManualToastDismissal);

function _showLifecycleToast(msgBody, msgType, requireManualToastDismissal, msgTitle) {
let finalMsgTitle = msgTitle || lifecycleStatusMessageTitle;
// use the args here to build the correct msgBody and options based on requireManualToastDismissal
}
```